CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+## 0.11.3
+
+### Preview features
+
+- \[`airflow`\] Update replacement paths for `AIR302` ([#16876](https://github.com/astral-sh/ruff/pull/16876))
+- \[`airflow`\] Add autofix for `AIR302` attribute checks ([#16977](https://github.com/astral-sh/ruff/pull/16977))
+- \[`airflow`\] Add autofix for `AIR302` method checks ([#16976](https://github.com/astral-sh/ruff/pull/16976))
+- \[`airflow`\] Add autofix infrastructure to `AIR302` name checks ([#16965](https://github.com/astral-sh/ruff/pull/16965))
```

Comment:
Some of those seem to be refactors only. We could collapse them into one change log entry saying Add more autofixes to `AIR302`